app/src/main/java/com/jerboa/MainActivity.kt

```diff
@@ -115,708 +118,710 @@ class MainActivity : AppCompatActivity() {
             JerboaTheme(
                 appSettings = appSettings,
             ) {
-                val appState = rememberJerboaAppState()
+                Surface(modifier = Modifier.fillMaxSize()) {
```

Comment:
Only change here is wrapping everything in this surface: its necessary for the navigation bar scaffold, and is in most of the examples.